tests/test_models.py

```diff
@@ -678,15 +678,16 @@ def test_string_conversion(self, method):
         Test ``EnterpriseCustomerPluginConfiguration`` conversion to string
         """
         channel = EnterpriseIntegratedChannel(
-            id=1, name='CorporateLMS', data_type='course'
+            id=1, name='CorporateLMS', data_type='catalog'
         )
         faker = FakerFactory.create()
         customer_uuid = faker.uuid4()
         plugin_config = EnterpriseCustomerPluginConfiguration(
             enterprise_customer_uuid=customer_uuid, enterprise_integrated_channel=channel, active=False
         )
-        expected_to_str = "<EnterpriseCustomerPluginConfiguration for enterprise {uuid} using channel CorporateLMS>"\
-            .format(uuid=customer_uuid)
+        expected_to_str = "<EnterpriseCustomerPluginConfiguration for enterprise {} using channel CorporateLMS" \
+                          " for data type catalog>"\
+            .format(customer_uuid)
```

Comment:
I'm not thrilled about the line joining format here. I'd prefer to use Python's native multiline string joining like so:

```python
expected_to_str = (
    "<EnterpriseCustomerPluginConfiguration for enterprise {} using "
    "channel CorporateLMS for data type catalog>"
).format(customer_uuid)